tutorindigo/patches/mfe-env-config-runtime-definitions

```diff
@@ -0,0 +1,4 @@
+
+
+const Footer = await import(/* webpackIgnore: true */`@edly-io/indigo-frontend-component-footer`).catch(() => null); // Fallback to null if not available
```

Comment:
```suggestion
const { default: IndigoFooter } = await import('@edly-io/indigo-frontend-component-footer');
```

First: instead of having to use ` Footer.default` below, it's less confusing to just rename `default` to the component we want.  (I suggest `IndigoFooter` to avoid confusion.)

Second: as far as I can tell, `webpackIgnore` will make it so webpack never imports the footer component, so we can't use that.  And I'm not convinced a `catch` would work as intended, here.  This is already in a try/catch block, which is the only way to get Webpack to actually do an optional import [as of v5.90.2](https://github.com/webpack/webpack/discussions/18027#discussioncomment-10073944).

Now, this does indeed break the build in frontend-app-ora-grading: webpack is not generating code that treats the import as optional.  I tracked this down to the fact this MFE is not configuring `browserslist` properly in package.json (see [PR that fixes it](https://github.com/openedx/frontend-app-ora-grading/pull/379)).  We'll fix the MFE, but I have a question in the meantime: any particular reason why is the Indigo footer not being npm-installed for frontend-app-ora-grading?

Reply:
> any particular reason why the Indigo footer is not being npm-installed for frontend-app-ora-grading?

Adding IndigoFooter to ORA requires styles from `indigo-brand-openedx` package (otherwise indigo-footer doesn't look appealing). And if we apply `indigo-brand-openedx` package, then dark-theme automatically applies  to `body` and it shows inconsistent, unreadable page of ora-grading MFE. We have planned to work on ora-grading styling after sumac. We'll include at that time.

Reply:
Got it, thanks!

I'm hoping to get the ORA grading MFE fixed today (including in the Sumac branch).